front/src/modules/rollingStock/components/RollingStockCard/RollingStockCardButtons.tsx

```diff
@@ -4,14 +4,12 @@ import { useTranslation } from 'react-i18next';
 
 import { ModalContext } from 'common/BootstrapSNCF/ModalSNCF/ModalProvider';
 import OptionsSNCF, { Option } from 'common/BootstrapSNCF/OptionsSNCF';
-import {
-  updatePathfindingID,
-  updatePowerRestrictionRanges,
-  updateRollingStockComfort,
-  updateRollingStockID,
-} from 'reducers/osrdconf';
-import { getRollingStockComfort } from 'reducers/osrdconf/selectors';
-import { useOsrdConfContext } from 'common/osrdConfContext';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { Comfort } from 'common/api/osrdEditoastApi';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { Comfort } from 'common/api/osrdEditoastApi';
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done